### ISSUE TITLE
Clean up some of the overlap in terms of concierge welcome, onboarding tasks and Guides messages

### ISSUE THREAD

**Issue by anmurali** [Expensify/App#41451]
Now that we have onboarding tasks in New Dot for a workspace admin, there's a bunch of overlapping messages from their Guide as well making things seem quite noisy. Laying out a few things to clean up here:

# First off, let's clean up the existing Concierge greeting. 

For half the sign ups - the onboarding tasks will remain in Concierge and the other half will go to Expensify system DM. But currently Concierge is already greeting every user, and then pointing workspace owner sign ups to Guides. 
![image](https://github.com/Expensify/App/assets/14225673/2425dab2-0bc1-47ab-a1db-da9d1e712f78)

- ## Let's get rid of this message altogether
>Let's get your company set up! , your dedicated specialist, is online now and can answer your initial questions or provide a demo.
>
>💬 CHAT WITH YOUR SETUP SPECIALIST

- ## Let's also get rid of this opening message on the onboarding tasks
>Hey @muralidharan.anuradha+stage2@gmail.com 👋

Let's replace that with:
>Let's get you setup (maybe a spanner emoji?)

Followed as is today by 
>Here are some important tasks....

# Next, the message from a Guide to

**Comment by anmurali** [comment]
No updates yet. We need to get this assigned.

**Comment by trjExpensify** [comment]
Asked @NikkiWines [here](https://expensify.slack.com/archives/C036QM0SLJK/p1715818774441159?thread_ts=1715336603.190319&cid=C036QM0SLJK) if she might be interested in this while we're holding on the phase2 App PR.

**Comment by NikkiWines** [comment]
Tackling this today!

**Comment by NikkiWines** [comment]
Just to confirm, we want the following for the updated onboarding message from `Hey @muralidharan.anuradha+stage2@gmail.com 👋` to `Let's get you set up 🔧`

![Screenshot 2024-05-20 at 17 20 36](https://github.com/Expensify/App/assets/16747822/37c4b276-d2c5-4989-bee1-b6295e9a7651)

And want to remove both of the following messages entirely - regardless where the user was onboarded from? 

```
Let's get your company set up! , your dedicated specialist, is online now and can answer your initial questions or provide a demo.

💬 CHAT WITH YOUR SETUP SPECIALIST
``` 

and 

```
Hi there 👋, I'm your dedicated setup specialist. I look forward to helping you explore and configure Expensify. A few important things to mention:

• You're on a 7-day free trial, but I can help extend that if you're interested.
• Your workspace has a ton of custom settings, just select your workspace settings to set it up.
• You've got more functionality to enable, like custom categories, tags, etc. Just ask me how.

Chat with me here in your #admins room or just reply to this message. You can also schedule a call if you have more in-depth questions. Talk soon!
```

**Comment by anmurali** [comment]
You got it @NikkiWines

**Comment by NikkiWines** [comment]
PRs are in review to update this

**Comment by trjExpensify** [comment]
>And want to remove both of the following messages entirely - regardless where the user was onboarded from?

Hmm, if they are onboarded via OldDot they don't get setup tasks in NewDot do they, so how will they be aware of their assigned Guide if we kill the Guides message?

**Comment by mountiny** [comment]
We are updating the code now to make sure only the newDot organic sign ups will be presented with the flow

**Comment by NikkiWines** [comment]
> Hmm, if they are onboarded via OldDot they don't get setup tasks in NewDot do they, so how will they be aware of their assigned Guide if we kill the Guides message?

Sounds like we only want to remove those messages if the users are being onboarded via newDot then. oldDot should stay as it is? Is that correct @trjExpensify?

**Comment by trjExpensify** [comment]
Yep, that's correct! Otherwise they have no setup tasks (and thus no surfacing of their assigned guide).

**Comment by trjExpensify** [comment]
^^ false flag :) 

Two PRs for this are deployed now:

- https://github.com/Expensify/App/pull/42408 (Prod)
- https://github.com/Expensify/Web-Expensify/pull/42109 (staging)

When the latter gets to prod, we're done here I believe.

**Comment by trjExpensify** [comment]
We can close this, right @NikkiWines?

**Comment by NikkiWines** [comment]
Yep, this can be closed!
